src/pages/workspace/companyCards/WorkspaceCompanyCardsListHeaderButtons.tsx

```diff
@@ -41,6 +44,32 @@ function WorkspaceCompanyCardsListHeaderButtons({policyID, selectedFeed}: Worksp
     const isCustomFeed = CardUtils.isCustomFeed(selectedFeed);
     const companyFeeds = CardUtils.getCompanyFeeds(cardFeeds);
     const currentFeedData = companyFeeds?.[selectedFeed];
+    const policy = PolicyUtils.getPolicy(policyID);
+
+    const [list] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${workspaceAccountID}_${selectedFeed}`);
+    const filteredCardList = CardUtils.getFilteredCardList(list);
+
+    const handleAssignCard = () => {
+        const data: Partial<AssignCardData> = {
+            bankName: selectedFeed,
+        };
+
+        let currentStep: AssignCardStep = CONST.COMPANY_CARD.STEP.ASSIGNEE;
+
+        if (Object.keys(policy?.employeeList ?? {}).length === 1) {
+            data.email = Object.keys(policy?.employeeList ?? {}).at(0);
+            currentStep = CONST.COMPANY_CARD.STEP.CARD;
+
+            if (CardUtils.hasOnlyOneCardToAssign(filteredCardList)) {
+                currentStep = CONST.COMPANY_CARD.STEP.TRANSACTION_START_DATE;
+                data.cardNumber = Object.keys(filteredCardList).at(0);
+                data.encryptedCardNumber = Object.values(filteredCardList).at(0);
```

Comment:
@koko57 I believe that the two fields are different

<img width="413" alt="Screenshot 2024-11-22 at 10 14 44" src="https://github.com/user-attachments/assets/aebcdf8e-b37d-4a2f-ac3a-42e79dc76693">

Reply:
what do you mean?

Reply:
you have the key - "480801XXXXX2554" for the card name and the value "v11:47A..." for the encryptedCradNumber

Reply:
@koko57 In your code change, you set two fields to the same value 
```
                data.cardNumber = Object.keys(filteredCardList).at(0);
                data.encryptedCardNumber = Object.values(filteredCardList).at(0);
```

Reply:
Object.keys vs  Object.values? how they are the same

Reply:
I'm getting the first key "480801XXXXX2554" and it's value "v11:47A..." f

Reply:
Sorry, my bad

Reply:
It is fine